ManageRDSessions/ManageRDSessions.psm1

```diff
@@ -1,7 +1,7 @@
 #Requires -RunAsAdministrator
 function Get-sbRDSession {
     [OutputType('Custom.SB.RDSession')]
-    [cmdletbinding(DefaultParameterSetName = 'State')]
+    [cmdletbinding(DefaultParameterSetName = 'State', HelpUri = "https://bit.ly/2xghizR")]
```

Comment:
I'd use full paths rather than redirect URLs here

Reply:
Removed in favour of comment-based help